testdata/project-v2/Makefile

```diff
@@ -54,7 +54,7 @@ generate: controller-gen
 docker-build: test
 	docker build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
+        cd config/manager && kustomize edit set image controller=${IMG}
```

Comment:
Indentation must be with tabs, not spaces.

Reply:
updated it. thx